### ISSUE TITLE
Release into foxy

### ISSUE THREAD

**Issue by agutenkunst** [klintan/ros2_usb_camera#13]
Hi @klintan what do you think about releasing this package?

**Comment by agutenkunst** [comment]
Update: Just found https://github.com/flynneva/usb_cam/tree/ros2 not sure about the differences.

**Comment by klintan** [comment]
Yes I agree! I'll try to do it this weekend :) I'll keep you posted.

**Comment by agutenkunst** [comment]
Also see https://discourse.ros.org/t/usb-cam-orphaned-offer-to-support/19367

**Comment by klintan** [comment]
Started this one now https://github.com/ros/rosdistro/pull/28637 I might have messed something up but hopefully it's release soon :D
